agent/consul/rpc.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/hashicorp/consul/agent/consul/state"
 	"github.com/hashicorp/consul/agent/metadata"
 	"github.com/hashicorp/consul/agent/pool"
+	"github.com/hashicorp/consul/agent/proto"
```

Comment:
Could you rename this to something like `agentproto` or `agentpb` so that it doesn't share a short package name with `github.com/gogo/protobuf/proto` which is also used in this repo?

Reply:
I certainly could but whats the advantage over just importing the package like:

```
import (
   gogoproto "github.com/gogo/protobuf/proto"
   agentproto "github.com/hashicorp/consul/agent/proto"
)
```

When you have two packages from different libs with the same name imported in the same file you have to differentiate like this. But in the simple case where you only need one or the other (which is most places) I don't see the problem with importing the regular name.

Reply:
The only place I can see them both being used simultaneously is in the `agent/xds` package.

I guess there is no harm in changing the name to make the auto-import addition dev workflow easier.